ui/components/privacy-view.jsx

```diff
@@ -0,0 +1,145 @@
+         These session cookies are set by GSA or a contractor acting on GSA’s behalf and let the server hosting the OMB Policy Library know that you are continuing a visit to our site.
+         Our use of session cookies qualifies as “Usage Tier 1—Single Session,” as defined in the OMB M-10-22 guidance.
+        Persistent Cookies: We use persistent cookies to understand the differences between new and returning visitors to the OMB Policy Library.
+        Persistent cookies remain on your device between visits to our site until they expire or are removed by the user.
+         As discussed below under “Google Analytics,” our use of persistent cookies qualifies as “Usage Tier 2—Multi-session without personally identifiable information,” as defined in the OMB M-10-22 guidance.
+         The policy states, “This tier encompasses any use of multi-session Web measurement and customization technologies when no [personally identifiable information] is collected.” We do not use persistent cookies to collect personally identifiable information.
+        If you do not want to accept cookies, you can edit your browser’s options to stop accepting persistent cookies or to prompt you before accepting a cookie from the websites you visit.
+         Here are instructions for how you can disable cookies and/or Google Demographic and Interests reports.
+         {/*eslint-enable */}
+      </p>
+
+
+      <h4>Google Analytics</h4>
+      <p className="max-width-3">
+        {/*eslint-disable */}
+        Fpc.gov participates in the U.S.
+         Digital Analytics Program, (DAP) which utilizes a unified Google Analytics account for Federal Government agencies.
+         This program helps Federal agencies understand how people find, access, and use government services online.
+
+        The DAP is a hosted shared service provided by the General Services Administration’s (GSA’s) Office of Citizen Services and Innovative Technologies, and the protocol and information collected are the same for all websites participating in the DAP.
+        As a participant in GSA’s DAP program, this website’s Google Analytics traffic data is automatically reported to GSA.
+
+        Google Analytics is a third-party web measurement and customization technology as defined in OMB M-10-22 (PDF).
+
+        Here is how it works: Google Analytics sets one or more cookies on your computer so that it can recognize your computer if you visit the the OMB Policy Library website in the future.
+         This cookie(s) does not collect personally identifiable information.
+         This is considered a Tier 2 usage, as defined in the OMB guidance.
+
+        Google Analytics does not collect personally identifiable information through its cookie(s).
+         The program does not track individuals and anonymizes the IP addresses of visitors.
+        Common Questions about DAP (FAQ) provides more information about how Google Analytics handles IP addresses.
```

Comment:
For the record, I'm only marking missing links when the text obviously indicates there should be a link. Otherwise I'm not noting all the ones in the fpc privacy policy.